rules/BUILD.std_java_runtime.tpl

```diff
@@ -20,15 +20,5 @@ package(default_visibility = ["//visibility:public"])
 
```

Comment:
BUILD.std_java.tpl

Reply:
same for the cc one, maybe BUILD.std_cc.tpl

Reply:
renamed